04. 완전탐색(블루투포스)/02. 뒤집은 소수/이서인/index.js

```diff
@@ -1,7 +1,20 @@
 function solution(arr) {
-  let answer = [];
-
-  return answer;
+  const reversedNumbers = arr.map((el) =>
+    parseInt([...(el + "")].reverse().join(""))
+  );
+  return reversedNumbers
+    .map((number) => {
+      if (number > 1) {
+        for (let i = 2; i <= Math.sqrt(number); i++) {
+          if (number % i === 0) {
+            return 0;
+          }
+        }
+        return number;
+      }
+    })
+    .filter((n) => n !== 0)
```

Comment:
저와 풀이가 비슷해요~ map을 거치지 않고 바로 filter를 해도 괜찮을 것 같아요~
```suggestion
    .filter((number) => {
      if (number > 1) {
        for (let i = 2; i <= Math.sqrt(number); i++) {
          if (number % i === 0) {
            return false;
          }
        }
        return true;
      }
      return false;
    })
```

Reply:
와웅 반영했어요 제 코드 꼼꼼히 봐주셔서 항상 감사합니당